.circleci/config.yml

```diff
@@ -21,20 +21,34 @@ executors:
       LEIN_ROOT: "true"   # we intended to run lein as root
       JVM_OPTS: -Xmx3200m # limit the maximum heap size to prevent out of memory errors
     <<: *defaults
+
+  # JDK 8 is preferred for deployments (https://github.com/benedekfazekas/mranderson/issues/57)
+  # Parallelism is disabled, at least for now (https://github.com/benedekfazekas/mranderson/issues/56)
+  openjdk8_deploy:
+    docker:
+      - image: circleci/clojure:openjdk-8-lein-2.9.1-node
+    environment:
+      LEIN_ROOT: "true"   # we intended to run lein as root
+      LEIN_JVM_OPTS: -Dmranderson.internal.no-parallelism=true
```

Comment:
Note that parallelism is disabled only in deployments, and because they're performed on CI, the slowness don't matters as much - one doesn't have to stare at the output 😄

Reply:
you might want to add a comment about this in the config.

Reply:
There are some comments in L25